specification/ninjs-schema_dev_2.0.json

```diff
@@ -2,6 +2,7 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "id": "http://www.iptc.org/std/ninjs/ninjs-schema_draft_2.0.json#",
     "type": "object",
+    "name": "ninjs",

```

Comment:
Check that this is valid JSON Schema 4 - it's required by the JSON Schema -to-protobufs generator tool.